File: Flatshare Code/FlatShare 2.0/FlatshareCentral/src/main/java/com/joinflatshare/ui/settings/SettingsActivity.java
package com.joinflatshare.ui.settings;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;

import com.joinflatshare.FlatshareCentral.databinding.ActivityFaqBinding;
import com.joinflatshare.ui.base.BaseActivity;
import com.joinflatshare.ui.faq.FaqAdapter;
import com.joinflatshare.utils.mixpanel.MixpanelUtils;

import java.util.ArrayList;

public class SettingsActivity extends BaseActivity {
    private ActivityFaqBinding viewBind;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        viewBind = ActivityFaqBinding.inflate(getLayoutInflater());
        setContentView(viewBind.getRoot());
        showTopBar(this, true, "Settings", 0, 0);
        new SettingsViewBind(this, viewBind);
        new SettingsListener(this, viewBind);
        MixpanelUtils.INSTANCE.onButtonClicked("FAQ");
    }
}

File: Flatshare Code/FlatShare 2.0/FlatshareCentral/src/main/java/com/joinflatshare/ui/settings/SettingsActivity.kt
package com.joinflatshare.ui.settings

import android.os.Bundle
import android.view.View
import com.joinflatshare.FlatshareCentral.databinding.ActivitySettingsBinding
import com.joinflatshare.constants.AppConstants
import com.joinflatshare.ui.base.BaseActivity
import com.joinflatshare.utils.mixpanel.MixpanelUtils
import com.joinflatshare.utils.mixpanel.MixpanelUtils.onButtonClicked

/**
 * Created by debopam on 21/05/24
 */
class SettingsActivity : BaseActivity() {
    private lateinit var viewBind: ActivitySettingsBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        viewBind = ActivitySettingsBinding.inflate(layoutInflater)
        setContentView(viewBind.root)
        MixpanelUtils.onScreenOpened("Settings")
        showTopBar(this, true, "Settings", 0, 0)
        SettingsViewBind(viewBind)
        SettingsListener(this, viewBind)
        checkAdminFeatures()
    }

    private fun checkAdminFeatures() {
        if (!AppConstants.isAppLive && AppConstants.isAdmin)
            viewBind.cardProfileAdmin.visibility = View.VISIBLE
    }
}